### DISCUSSION TITLE
Suggest a different mapping for `coc-implementation`

### DISCUSSION THREAD

**Discussion by pharmpy-dev-123** [neoclide/coc.nvim#4051] [Q&A]
The current suggestion `nmap <silent> gi <Plug>(coc-implementation)` clashes with the very convenient default `gi` ([jump in insertion mode at last insertion mode position](https://vimhelp.org/insert.txt.html#gi)). What would be a better suggestion for this mapping? `gl`? See *2.4 Commands starting with 'g'* in https://vimhelp.org/index.txt.html.

**Comment by chemzqm** [comment]
PR welcome.
